lib/login.dart

```diff
@@ -1,28 +1,79 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/flutter_svg.dart';
+import 'package:logger/logger.dart';
+import 'package:appwrite/appwrite.dart';
+import 'package:appwrite/models.dart' as models;
+
 import 'package:langx_flutter/dfault_framework/dfault_material/dfault_buttons.dart';
 import 'package:langx_flutter/dfault_framework/dfault_material/dfault_input.dart';
-import 'package:langx_flutter/pages/home/head_bar.dart';
+
+// Initialize Logger
+var logger = Logger();
 
 class LoginScreen extends StatefulWidget {
-  const LoginScreen({super.key});
+  const LoginScreen({super.key, required this.account});
+
+  final Account account;
 
   @override
   LoginScreenState createState() => LoginScreenState();
 }
 
 class LoginScreenState extends State<LoginScreen> {
   final _formKey = GlobalKey<FormState>();
+  final TextEditingController emailController = TextEditingController();
+  final TextEditingController passwordController = TextEditingController();
+  final TextEditingController nameController = TextEditingController();
+  models.User? loggedInUser;
+
+  Future<void> login(String email, String password) async {
+    try {
+      await widget.account
+          .createEmailPasswordSession(email: email, password: password);
+      final user = await widget.account.get();
+      setState(() {
+        loggedInUser = user;
+      });
+
+      // Log the user
+      logger.i('logged in: $loggedInUser');
+    } catch (e) {
+      // Handle error
+      logger.e('Login failed: $e');
```

Comment:
We can do instead debugprint(e) or just print(e), but no issue if using logger :)

Reply:
print e works for production as well but logger works for only in development 

for print, flutter analyze gives warnings as well